ert_shared/tracker/evaluator.py

```diff
@@ -19,7 +19,8 @@ def __init__(self, model, ee_monitor_connection_details, states):
 
         host, port = ee_monitor_connection_details
         self._snapshot_mutex = threading.Lock()
```

Comment:
Since we're not guarding the snapshot anymore, maybe find a better name?

Reply:
Also, wouldn't this guard go away if you communicated the updates via the queue? Weren't you talking about doing this?